content/questions/admin/question-046.md

```diff
@@ -0,0 +1,11 @@
+---
+question: "How can a GitHub organization enhance its security posture?"
+archetype: "questions"
+title: "Question 046"
+draft: false
+---
+
```

Comment:
any link to this? it's a very broad question

how about everything that GHAS covers, that surely enhances the `security posture`